collector/bcache_linux.go

```diff
@@ -43,7 +43,7 @@ func NewBcacheCollector(logger log.Logger) (Collector, error) {
 
 	return &bcacheCollector{
 		fs:     fs,
-		logger: logger,
+		logger: log.With(logger, "collector", "bcache"),
```

Comment:
You shouldn't have to repeat this in every individual collector, do it once in NewNodeCollector